web/src/lib/components/shared-components/map/map.svelte

```diff
@@ -28,6 +28,14 @@
   export let zoom: number | undefined = undefined;
   export let center: LngLatLike | undefined = undefined;
   export let simplified = false;
+  export let clickable = false;
+
+  let map: maplibregl.Map;
+  let marker: maplibregl.Marker | null = null;
+
+  $: if (map) {
+    map.on('click', handleMapClick);
+  }
```

Comment:
You should have something like `on:load={(event) => event.detail.on('click', handleClick)}` on the MapLibre component instead.

Reply:
@YFrendo Can you fix this as well?